src/Castle.Windsor/Windsor/IWindsorContainer.cs

```diff
@@ -157,17 +157,9 @@ IWindsorContainer AddFacility<TFacility>(Action<TFacility> onCreate)
 		///   Returns a component instance by the service
 		/// </summary>
 		/// <param name = "service"></param>
-		/// <param name = "arguments"></param>
+		/// <param name = "arguments">Arguments to resolve the service. Please see the following factory methods: <see cref="Arguments.FromDictionary"/>, <see cref="Arguments.FromProperties"/>, <see cref="Arguments.FromReadOnlyDictionary"/> and <see cref="Arguments.FromTypedArguments"/></param>
```

Comment:
I'd prefer not to have to maintain a hardcoded list of all the factory methods in the XML docs on each IWIndsorContainer overload, rather just mention the static factory methods of the Arguments class.